src/app/hooks/useCommandPalette.ts

```
@@ -0,0 +1,201 @@
+    switch (e.key) {
+      case 'ArrowDown':
+        e.preventDefault();
+        setActiveIndex(i => (i + 1) % searchResults.length);
+        break;
+      case 'ArrowUp':
+        e.preventDefault();
+        setActiveIndex(i => (i - 1 + searchResults.length) % searchResults.length);
+        break;
+      case 'Enter':
+        e.preventDefault();
+        const selectedResult = searchResults[activeIndex];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)